### PR TITLE
Carousels

### PR DESCRIPTION
**Pull request by mxsnyder**
Huzzah! Another pull request to review. This one's for the Animal Services carousels. This is actually already implemented in staging , it just needs to be merged. You can see it in action here:

[http://staging.baltimorecountymd.gov/animal-services-redesign/index-copy.html](http://staging.baltimorecountymd.gov/animal-services-redesign/index-copy.html)

### CODE REVIEW

File: js/internal-carousel.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+namespacer('baltimoreCounty');
+
+baltimoreCounty.internalCarousel = (function($) {
+
+	var fixLinks = function() {
+			var $links = $('.carousel selink');
+
+			$links.each(function(index, selink) {
+				var $selink = $(selink),
```
Comment (martypowell) [inline js/internal-carousel.js]: Use var instead of commas

File: js/internal-carousel.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+					href = $selink.attr('externallink') ? $selink.attr('externallink') : $selink.attr('href'),
+					$newLink = $('<a>').attr('href', href).attr('title', $selink.attr('title')).text($selink.text());
+
+				if ($selink.attr('newwindow') && $selink.attr('newwindow') != 'false')
```
Comment (martypowell) [inline js/internal-carousel.js]: Include brackets on conditional

File: stylesheets/partials/homepage/_carousel.scss [hunk 3]
```diff
@@ -1,170 +1,184 @@
...
-
-  opacity: 0.25;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+	position: absolute;
+	top: 0;
+	right: 50px;
+	width: 20px;
+	height: 20px;
+	font-family: "slick";
```
Comment (martypowell) [inline stylesheets/partials/homepage/_carousel.scss]: I wonder if we should store this font in variable

File: js/internal-carousel.js (hunk 1 above)
Comment (mxsnyder) [inline js/internal-carousel.js]: Done!

File: js/internal-carousel.js (hunk 2 above)
Comment (mxsnyder) [inline js/internal-carousel.js]: Done!

File: stylesheets/partials/homepage/_carousel.scss (hunk 3 above)
Comment (mxsnyder) [inline stylesheets/partials/homepage/_carousel.scss]: Done!

**Pull request merged**